interview-questions.md

```diff
@@ -6,36 +6,36 @@ First, without external resources. Challenge yourself to answer from memory as i
 
 Then, research the question to expand on your answer. Even if you feel you have answered the question completely, there is always something more to learn. Write your researched answer in your OWN WORDS.
 
-Type your initials here to confirm you read the instructions:
+Type your initials here to confirm you read the instructions: JR
 
 1. What is the difference between a parameter and an argument?
 
-Your answer:
+Your answer: A parameter is the name for the word that is like a placeholder, for example in a function the word inside the parentheses is a parameter. It usually describes what type of arguement should be passed in, if the function required a number to be passed in the word number would be typed in to the parentheses to let the person know that a number should be the arguement. The arguement itself is what actually gets put into that spot whether it is the correct data type or not. In the same example of a function the arguemnet is what is passed into the paranetheses. In our case it says number, but a string can be passed into it even though it may produce an error. The string in this case would be the argument even though the parameter is showing that a number arguement should be used in our example.
 
-Researched answer:
+Researched answer: A parameter holds no value, it's essentially a placeholder for an arguement. An arguement is the value passed into the function when it is invoked taking the place of the parameter.
 
-2. What is the difference between an object and a class?
+2. What is the difference between an object and a class? 
+Your answer: An object is a variable that can store multiple pieces of information about that object in what we call key value pairs. They are easily accesed by outputting the name of the object followed by a period then the name of the key. A class is a would be a collection of objects that are alike. An example of the differences can be Mazda, Ford, Ferrari, and Toyota can be objects, each of them holding values of different data like the model name, color, and year. The class in this example would be named something in common with all of these. We can name it Cars, we differintiate them becasue classes will always begin with a capital letter and objects would be spelled in camelCase.
```

Comment:
I like the use of examples here. However, a class is not a collection of objects, it is more of a recipe to create objects with. You can create objects from the class, but the class does not hold objects.